src/utils/user-role-utils.tsx

```diff
@@ -0,0 +1,34 @@
+import { useAtomValue } from "jotai";
+import { authAtom } from "../atoms/auth";
+import { useLocation } from "react-router";
+
+/**
+ * User Role Utils class
+ */
+export default class UserRoleUtils {
+  /**
+   * Check if the logged-in user has an admin role
+   *
+   * @param accessToken keycloak access token
```

Comment:
It seems this function no longer takes a parameter?

Reply:
Yes, my mistake